pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -140,7 +140,7 @@ self: super: builtins.intersectAttrs super {
 
   hs-mesos = overrideCabal (drv: {
```

Comment:
Just delete the entire override and add `hs-mesos` to `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml`, please.

Reply:
Done

Reply:
Do I need to regenerate the package set?

Reply:
Up to you, doesn't really matter; it'd be done eventually after a couple of days.